src/app/shared/notification/service/notification.service.ts

```diff
@@ -25,6 +25,14 @@ export class NotificationService {
     this.toastrService.info(announcementNotification, 'Just to let you know!', options);
   }
 
+  private addComplexNavigatorAnnouncementNotification(announcement: string): void {
+    this.toastrService.toastrConfig.disableTimeOut = true;
```

Comment:
I think when we do `this.toastrService.toastrConfig.disableTimeOut = true;`, we're changing the config for all toasts, not just for this announcement, so if any other toast gets displayed, it will also not have the timeout.

The way to do it only for this is
```
    return this.toastrService.info(announcement, null, {
      disableTimeOut: true,
      closeButton: true,
      progressBar: false,
      positionClass: 'toast-right-under-header'
    });
```
2nd argument is null as it is the title, which we do not set.